components/brave_wallet/common/eth_sign_typed_data_helper_unittest.cc

```diff
@@ -32,26 +32,59 @@ TEST(EthSignedTypedDataHelperUnitTest, EncodeTypes) {
                                              base::JSON_ALLOW_TRAILING_COMMAS);
   ASSERT_TRUE(types_value);
 
-  std::unique_ptr<EthSignTypedDataHelper> helper =
+  std::unique_ptr<EthSignTypedDataHelper> valid_types_helper =
       EthSignTypedDataHelper::Create(types_value->GetDict().Clone(),
                                      EthSignTypedDataHelper::Version::kV4);
-  ASSERT_TRUE(helper);
-  const std::string encoded_types_v4 = helper->EncodeTypes("Mail");
+
+  ASSERT_TRUE(valid_types_helper);
+  const std::string encoded_types_v4 = valid_types_helper->EncodeTypes("Mail");
   EXPECT_EQ(encoded_types_v4,
             "Mail(Person from,Person to,string contents)Person(string "
             "name,address wallet)");
-  auto typed_hash_v4 = helper->GetTypeHash("Mail");
+  auto typed_hash_v4 = valid_types_helper->GetTypeHash("Mail");
   EXPECT_EQ(base::ToLowerASCII(base::HexEncode(typed_hash_v4)),
             "a0cedeb2dc280ba39b857546d74f5549c3a1d7bdc2dd96bf881f76108e23dac2");
 
   // v3 should be same as v4
-  helper->SetVersion(EthSignTypedDataHelper::Version::kV3);
-  const std::string encoded_types_v3 = helper->EncodeTypes("Mail");
+  valid_types_helper->SetVersion(EthSignTypedDataHelper::Version::kV3);
+  const std::string encoded_types_v3 = valid_types_helper->EncodeTypes("Mail");
   EXPECT_EQ(encoded_types_v4, encoded_types_v3);
-  auto typed_hash_v3 = helper->GetTypeHash("Mail");
+  auto typed_hash_v3 = valid_types_helper->GetTypeHash("Mail");
```

Comment:
can we remove these unnecessary  changes?

Reply:
Done here: [76342ce](https://github.com/brave/brave-core/pull/16883/commits/76342ce136c43e1af0dc2a07bd2c126a6449b990)

I ended up hitting some issues when just trying to drop the commit and when I tried to just revert it, so I just squashed all the changes back together since I figured I'd need to do that anyways. Hopefully that doesn't make it too much harder for you to review.